operator/v1/types_etcd.go

```diff
@@ -28,12 +28,34 @@ type Etcd struct {
 
 type EtcdSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+	// profile allows user to change the etcd tuning profile which configures
+	// the latency parameters for heartbeat interval and leader election timeouts
+	// allowing the cluster to tolerate longer round-trip-times between etcd members.
+	// Valid values are "Default" and "Slower".
+	// The current default is Default.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=Default
+	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +optional
+	Profile Profile `json:"profile,omitempty"`
 }
 
 type EtcdStatus struct {
 	StaticPodOperatorStatus `json:",inline"`
 }
 
+const (
+	// DefaultProfile sets Heartbeat interval: 100ms, Leader election timeout: 1000ms
+	DefaultProfile Profile = "Default"
+	// SlowerProfile sets Heartbeat interval: 500ms, Leader election timeout: 2500ms
```

Comment:
Describe the intent, not the values.  Perhaps as a multiple off the default.  That way values can be subject to change in the future.  You can say what they currently are, but they are subject to change without notice.